pages/blog/governance-motivation.md

```diff
@@ -0,0 +1,126 @@
+featured: true
+---
+
+> tl;dr charter for the AsyncAPI Initiative open governance is ready for the review. Leave your comments there in [this](https://github.com/asyncapi/.github/pull/37) pull request. We will wait two weeks for your feedback.
+
+December 2020 brought to AsyncAPI an important announcement, a [partnership with Postman](/blog/asyncapi-partners-with-postman). This huge milestone secured the AsyncAPI Initiative development efforts as few active community members moved to Postman. This move speeded up our work on transferring AsyncAPI to a foundation and forming an open governance model to assure the community that a single company does not control AsyncAPI Initiative.
+
+It took us a lot of time to write down the initial charter for the initiative.
+
+> **Pro Tip**:
+[Charter](https://dictionary.cambridge.org/dictionary/english/charter) is not a person that creates charts :smiley:
+Nobody's trying to be a smart ass here. We actually had to google that :smiley:
+
+## The vision
+
+We started with a basic vision in our heads and studied governance models of many diverse communities, like NodeJS Foundation, OpenJS Foundation, GraphQL Foundation, Cloud Events, OpenAPI Initiative, and CNCF. We also got a charter template from a foundation. That was a lot to handle. 
+
+<center><img className="w-1/3" src="https://media.giphy.com/media/5QMPpCzH6yxEqhev5A/giphy.gif" /></center>
+
+In the end, we just wanted to make sure the initiative is community-driven and leaves no man behind. We tried to figure out a governance model that:
+- is as democratic as possible
+- supports the asynchronous decision-making process
+- gives power to people that "work", not companies that "pay". In other words, it gives equal power to both individual and corporate contributors.
+
+### Vocabulary
+
+In the governance model, you can find two essential roles, contributor and committer. A contributor is a person that contributes to the project code, docs, or other artifacts. A committer is a person that contributes regularly and is invited by other committers to manage the repository, to have more privileges, and to approve pull requests.
+
+Committer rules in the repository, but for topics that go beyond, there is a Technical Steering Committee (TSC). 
+
+TSC consists of all committers. 
+
+Yes, not a dedicated group of people for now. Are you a committer? If yes, then it means you are a voting member of TSC (unless you do not want to). But more on that further in the blog post.
+
+### Single TSC for spec and tools
+
+Spec and tools will have different licenses, but there will not be two TSCs though. One TSC for now to handle both specification and tools.
+
+Why?
+
+- start small, they say.
+- don't design processes and then work accordingly but better build a process that facilitates the way you work already.
+- we are a large community, but not large enough to handle two different charters, not now.
```

Comment:
Did you mean `, not now` or rather `, for now` ?

Reply:
`not now`, there is a comma before so dramatically it should be correct